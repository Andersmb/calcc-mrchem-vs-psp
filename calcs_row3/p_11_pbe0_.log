Starting job 5688403 on c1-[24,45],c2-49,c10-13,c11-34 at Fri May 20 15:54:54 CEST 2022

Copying p_11_pbe0_.out to /cluster/home/ambr/projects/psp/row3/calcs/doublets
Copying p_11_pbe0_.json to /cluster/home/ambr/projects/psp/row3/calcs/doublets

Task and CPU usage stats:
JobID           JobName  AllocCPUS   NTasks     MinCPU MinCPUTask     AveCPU    Elapsed ExitCode 
------------ ---------- ---------- -------- ---------- ---------- ---------- ---------- -------- 
5688403      p_11_pbe0_        180                                             00:22:18      0:0 
5688403.bat+      batch         36        1   00:00:02          0   00:00:02   00:22:18      0:0 
5688403.ext+     extern        180        5   00:00:00          4   00:00:00   00:22:18      0:0 
5688403.0      mrchem.x        180       30   00:21:32         29   00:56:56   00:21:41      0:0 

Memory usage stats:
JobID            MaxRSS MaxRSSTask     AveRSS MaxPages   MaxPagesTask   AvePages 
------------ ---------- ---------- ---------- -------- -------------- ---------- 
5688403                                                                          
5688403.bat+     19276K          0     19276K        0              0          0 
5688403.ext+          0          4          0        0              4          0 
5688403.0      8118064K         29 3738477772        0             10          0 

Disk usage stats:
JobID         MaxDiskRead MaxDiskReadTask    AveDiskRead MaxDiskWrite MaxDiskWriteTask   AveDiskWrite 
------------ ------------ --------------- -------------- ------------ ---------------- -------------- 
5688403                                                                                               
5688403.bat+       19.41M               0         19.41M        0.06M                0          0.06M 
5688403.ext+        0.00M               4          0.00M            0                4              0 
5688403.0           7.32M              22          6.97M       67.01M               19         44.28M 

Job 5688403 completed at Fri May 20 16:17:13 CEST 2022
