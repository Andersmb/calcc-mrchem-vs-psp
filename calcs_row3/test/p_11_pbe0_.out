                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         HEAD                                     ***
***         Git commit hash    e7d4b230bb6df8495b9a                     ***
***         Git commit author  Stig Rune Jensen                         ***
***         Git commit date    Sat Mar 19 14:14:51 2022 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :        (1 bank)                              30
 OpenMP threads          :                                               6
 Total cores             :                                             175
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : MPI/OpenMP (6 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 9
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               2
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       O            :        0.000000        0.000000       -1.839074
    1       P            :        0.000000        0.000000        0.949972
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000       -0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-04
 Screening               : 1.20000e+01 StdDev
 Restricted              : False
 Functional              : LDA (SVWN5)
 AO basis                : 3-21G
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                              12
 Beta electrons          :                                              11
 Total electrons         :                                              23
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              9.999999927613e-01
    1    1    a          :                              1.000000003036e+00
    2    1    a          :                              9.999999945547e-01
    3    1    a          :                              1.000000142300e+00
    4    1    a          :                              1.000000011052e+00
    5    1    a          :                              1.000000011005e+00
    6    1    a          :                              9.999979167897e-01
    7    1    a          :                              9.999999840957e-01
    8    1    a          :                              1.000002547187e+00
    9    1    a          :                              1.000000001746e+00
   10    1    a          :                              1.000002647222e+00
   11    1    a          :                              9.999980055156e-01
   12    1    b          :                              9.999999927613e-01
   13    1    b          :                              9.999999958897e-01
   14    1    b          :                              9.999999945553e-01
   15    1    b          :                              1.000000001516e+00
   16    1    b          :                              1.000000011052e+00
   17    1    b          :                              1.000000011005e+00
   18    1    b          :                              9.999999464424e-01
   19    1    b          :                              1.000000001206e+00
   20    1    b          :                              1.000002547233e+00
   21    1    b          :                              1.000000000643e+00
   22    1    b          :                              1.000002647272e+00
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE0)
 Relativity              : None
 Precision               : 1.00000e-04
 Localization            : On
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                413.189890907995
 E-N energy              :            (au)              -1071.879987146796
 Coulomb energy          :            (au)                233.282297139024
 Exchange energy         :            (au)                 -7.731707912420
 X-C energy              :            (au)                -23.942386266936
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                 43.025455519551
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -457.081893279135
 Nuclear energy          :            (au)                 43.025455519551
---------------------------------------------------------------------------
 Total energy            :            (au)             -4.140564377596e+02
                         :      (kcal/mol)             -2.598243353945e+05
                         :        (kJ/mol)             -1.087105019996e+06
                         :            (eV)             -1.126704916031e+04
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -76.877592411403
    1    1    a          :            (au)                -18.743718398053
    2    1    a          :            (au)                 -6.733706944654
    3    1    a          :            (au)                 -4.876380987606
    4    1    a          :            (au)                 -4.873602754512
    5    1    a          :            (au)                 -4.870679465172
    6    1    a          :            (au)                 -0.935933529267
    7    1    a          :            (au)                 -0.553764269504
    8    1    a          :            (au)                 -0.359800436774
    9    1    a          :            (au)                 -0.260080968985
   10    1    a          :            (au)                 -0.237979437132
   11    1    a          :            (au)                 -0.090778256362
   12    1    b          :            (au)                -76.873919453562
   13    1    b          :            (au)                -18.738545866634
   14    1    b          :            (au)                 -6.729574750295
   15    1    b          :            (au)                 -4.874266262225
   16    1    b          :            (au)                 -4.868613217856
   17    1    b          :            (au)                 -4.859541034519
   18    1    b          :            (au)                 -0.921822262467
   19    1    b          :            (au)                 -0.518883361928
   20    1    b          :            (au)                 -0.253252895944
   21    1    b          :            (au)                 -0.228494362124
   22    1    b          :            (au)                 -0.215006318335
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -238.495937645312
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (PBE0)
 Relativity              : None
 Checkpointing           : Off
 Max iterations          : 200
 KAIN solver             : 10
 Localization            : First two iterations
 Diagonalization         : Off
 Start precision         : 1.00000e-06
 Final precision         : 1.00000e-06
 Helmholtz precision     : Dynamic
 Energy threshold        : 1.00000e-06
 Orbital threshold       : 5.00000e-05
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          4.795832e+00        -414.056437759584         -4.140564e+02
    1          3.797981e-01        -416.268654008088         -2.212216e+00
    2          1.938050e-01        -416.364184228247         -9.553022e-02
    3          1.115391e-01        -416.380990778307         -1.680655e-02
    4          5.325976e-02        -416.387451956671         -6.461178e-03
    5          4.847171e-02        -416.391278121210         -3.826165e-03
    6          1.894353e-02        -416.391816189425         -5.380682e-04
    7          1.127217e-02        -416.392115367141         -2.991777e-04
    8          6.584569e-03        -416.392177371585         -6.200444e-05
    9          3.549305e-03        -416.392196380573         -1.900899e-05
   10          1.883458e-03        -416.392202438417         -6.057843e-06
   11          1.427797e-03        -416.392205094757         -2.656340e-06
   12          8.793344e-04        -416.392205970015         -8.752578e-07
   13          5.415104e-04        -416.392206321554         -3.515391e-07
   14          4.744779e-04        -416.392206548772         -2.272185e-07
   15          4.597655e-04        -416.392206713880         -1.651078e-07
   16          2.871142e-04        -416.392206773633         -5.975284e-08
   17          1.402022e-04        -416.392206795866         -2.223317e-08
   18          6.908462e-05        -416.392206804038         -8.172151e-09
---------------------------------------------------------------------------
                      SCF converged in 18 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               2
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       O            :        0.000000        0.000000       -1.839074
    1       P            :        0.000000        0.000000        0.949972
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000       -0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                415.655961192276
 E-N energy              :            (au)              -1076.423740145342
 Coulomb energy          :            (au)                232.925892361489
 Exchange energy         :            (au)                 -7.700994376626
 X-C energy              :            (au)                -23.874781355386
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                 43.025455519551
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -459.417662323589
 Nuclear energy          :            (au)                 43.025455519551
---------------------------------------------------------------------------
 Total energy            :            (au)             -4.163922068040e+02
                         :      (kcal/mol)             -2.612900525873e+05
                         :        (kJ/mol)             -1.093237580735e+06
                         :            (eV)             -1.133060867116e+04
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -77.280999878195
    1    1    a          :            (au)                -19.246129875869
    2    1    a          :            (au)                 -6.676705966949
    3    1    a          :            (au)                 -4.814879266546
    4    1    a          :            (au)                 -4.813139394399
    5    1    a          :            (au)                 -4.808023791019
    6    1    a          :            (au)                 -1.057590730321
    7    1    a          :            (au)                 -0.579601632295
    8    1    a          :            (au)                 -0.429422120439
    9    1    a          :            (au)                 -0.411443595058
   10    1    a          :            (au)                 -0.410260207333
   11    1    a          :            (au)                 -0.217149637297
   12    1    b          :            (au)                -77.277580254581
   13    1    b          :            (au)                -19.239170622719
   14    1    b          :            (au)                 -6.673194060670
   15    1    b          :            (au)                 -4.812964676333
   16    1    b          :            (au)                 -4.806381055651
   17    1    b          :            (au)                 -4.800266517178
   18    1    b          :            (au)                 -1.043971999060
   19    1    b          :            (au)                 -0.550323667842
   20    1    b          :            (au)                 -0.407005321278
   21    1    b          :            (au)                 -0.392352938144
   22    1    b          :            (au)                 -0.384614712228
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -241.133171921404
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :        0.000000        0.000001        1.247163
 Magnitude               :            (au)                        1.247163
                         :         (Debye)                        3.169971
---------------------------------------------------------------------------
 Nuclear vector          :        0.000000        0.000000       -0.463008
 Magnitude               :            (au)                        0.463008
                         :         (Debye)                        1.176848
---------------------------------------------------------------------------
 Total vector            :        0.000000        0.000001        0.784155
 Magnitude               :            (au)                        0.784155
                         :         (Debye)                        1.993123
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h 21m 33s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
